spec/features/merchant/items/new_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+RSpec.describe "Merchant_items#new", type: :feature do
+  # US 11
+  # As a merchant
+  # When I visit my items index page
+  # I see a link to create a new item.
+  # When I click on the link,
+  # I am taken to a form that allows me to add item information.
+  # When I fill out the form I click ‘Submit’
+  # Then I am taken back to the items index page
+  # And I see the item I just created displayed in the list of items.
+  # And I see my item was created with a default status of disabled.
+  describe "as a visitor" do
+    describe "when I am taken to a new item form" do
+      let!(:merchant_1) { create(:merchant) }
+
+      it "renders a form to create a new item" do
+        visit new_merchant_item_path(merchant_id: merchant_1.id)
+        expect(page).to have_content("Create New Item")
+        fill_in "Name:", with: "coffee mug"
+        fill_in "Description:", with: "stainless travel 16oz"
+        fill_in "Unit Price:", with: "999"
+        fill_in "Status:", with: "disabled"
```

Comment:
hey @kbergstrom78  user story says that the default for the model should be "disabled", currently in your model, the default is "1"/enabled. But this user story says is "created with a default of "disabled"" so when I run the rails server, it autopopulates "disable".

can we update the migration to have a default of "disabled" so that this user story is correct?

Reply:
but otherwise, really great job with this!!!

Reply:
Noooooooooooo!!!  But yes, I will fix that.

Reply:
hey @kbergstrom78 I'm still seeing the default as enabled in the migration - have you pushed up the changes you made?

Reply:
hmmm.  let me check.

Reply:
Something didn't take.  I'm going to leave default status as "1" in the migration, but update the enum so 1 = disabled.  Hang on.

Reply:
Okay - just checked in the rails console, should be good now.  Thanks!